deploy/resources/addons/02-clustertasks/git-clone/git-clone-task.yaml

```diff
@@ -69,12 +69,16 @@ spec:
       description: git no proxy - opt out of proxying HTTP/HTTPS requests
       type: string
       default: ""
+    - name: gitInitImage
+      description: The image used where the git-init binary is.
+      default: "gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/git-init:v0.15.2"
```

Comment:
We need to replace this with `registry.redhat.io/openshift-pipelines-tech-preview/pipelines-git-init-rhel8`

Reply:
shall we handle that in the `deploy/operator.yaml` Override. (also in CSV)
https://github.com/openshift/tektoncd-pipeline-operator/blob/2f271f6761de79fb494c4c8a570486f6e1c28217/deploy/operator.yaml#L27

Reply:
We tried to handle in script for this like in case of buldah, s2i and others.

Reply:
we dont need to replace,it will be handled by csv